docs/Algorithms.md

```diff
@@ -1,8 +1,35 @@
 # Implemented Compression Methods
 
-Each compression method receives its own hyperparameters that are organized as a dictionary and basically stored in a JSON file that is deserialized when the training starts. Compression methods can be applied separately or together producing sparse, quantized, or both sparse and quantized models. For more information about the configuration, refer to the samples.
+## Post-training Compression
 
-- [Quantization](./compression_algorithms/Quantization.md)
+- [Quantization](./usage/post_training_compression/post_training_quantization/Usage.md) (OpenVino, PyTorch, ONNX, TensorFlow)
+  - Symmetric and asymmetric quantization modes
+  - Signed and unsigned
+  - Per tensor/per channel
+  - Each backend support export to the OpenVino format
+- [Weights compression](./usage/post_training_compression/weights_compression/Usage.md) (OpenVino, PyTorch)
+  - Symmetric 8 bit compression mode
+  - Symmetric and asymmetric 4 bit compression mode
+  - NF4 compression mode
+  - Mixed precision weights compression
+  - Grouped weights compression
+
+## Training Time Compression
+
+### Quantization
+
+- [Quantization](./usage/training_time_compression/quantization_aware_training/Usage.md) (PyTorch)
```

Comment:
Maybe it's better to call it as Quantization Aware Training (QAT).
@alexsu52 what do you think?

Reply:
I agree with you.